### DISCUSSION TITLE
Clarity around the trigger_id field on WarrantExercise.schema.json

### DISCUSSION THREAD

**Discussion by jacobyavis** [Open-Cap-Table-Coalition/Open-Cap-Format-OCF#247] [General]
The current description wasn't enough to help me understand what this is supposed to be exactly. Perhaps my limited knowledge around warrants in general is also to blame

**Comment by pjohnmeyer** [comment]
The `trigger_id` in WarrantExercise references one of the triggers defined in the corresponding WarrantIssuance.
